app/graphql/types/concerns/profile_pseudo_policy.rb

```diff
@@ -5,18 +5,21 @@ module Types
   module ProfilePseudoPolicy
     extend ActiveSupport::Concern
 
+    # Use policy name for a policy profile
+    def name
+      object == policy ? object.policy_object.name : object.name
+    end
+
+    # Use policy name for a policy profile
```

Comment:
Gotta say, I really hate comments that describe what the code is doing when it's obvious. Even more so when they describe what the code isn't doing.

```suggestion
    # Use policy description for a policy profile
```

Reply:
My bad. :smile: